numba_dpex/tests/test_examples.py

```diff
@@ -11,6 +11,6 @@
 
 def test_examples_available():
     package_path = os.path.dirname(numba_dpex.__file__)
-    examples_path = os.path.join(package_path, "examples")
+    examples_path = os.path.join(package_path, "../examples")
```

Comment:
Same issue as the debugging test. Once the tests are out of tree, I suggest we remove this particular unit test completely.

Reply:
Removed tests_examples.py file from numba_dpex/tests.